dev.pulsar-edit.Pulsar.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.pulsar-edit.Pulsar",
```

Comment:
This app-id isn't valid. as per the documentatin, `-` are not allowed in there except for the last component and must be replaced by `_`.

This applied everywhere the app-id is used.